corehq/apps/users/static/users/js/mobile_workers.ng.js

```diff
@@ -94,6 +96,20 @@
         $scope.customFormFields = customFields;
         $scope.customFormFieldNames = customFieldNames;
 
+        $scope.availableLocations = [];
+
+        $scope.searchLocations = function (query) {
+            var reqStr = location_url + "?name=" + query;
+            $http.get(reqStr).then(
+                function (response) {
+                    $scope.availableLocations = response.data;
+                },
+                function () {
+                    console.log('Error retrieving locations');
```

Comment:
did you mean to leave this in? if so, should be `console.err` or include more information about the error?